src/main/java/frc/robot/subsystems/swerve/modules/factory/MK4IModuleContainer.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.subsystems.swerve.modules.factory;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.FeedbackSensorSourceValue;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import frc.robot.subsystems.swerve.modules.ModuleID;
+import frc.robot.subsystems.swerve.modules.mk4i.MK4IModuleConstants;
+
+public class MK4IModuleContainer {
+
+    private static final double DRIVE_SLIP_CURRENT = 30; //todo - calibrate
+    private static final double STEER_CURRENT_LIMIT = 30; //todo - calibrate
```

Comment:
if neds to calibrate, consider changing to `Double.POSITIVE_INFINITY`

Reply:
when i run swerve i keep the current limit to battery